angular/projects/admin-nrpti/src/app/records/orders/order-add-edit/order-add-edit.component.html

```diff
@@ -27,8 +27,10 @@ <h4 class="blue-header">Shared Data [Master]</h4>
           <div class="row">
             <div class="col-7"></div>
             <div class="col-5">
-              <span class="grey-subtext">If content is published on any site, hitting the Save/Update
-                Record button will update the public site immediately</span>
+              <span class="grey-subtext"
+                >If content is published on any site, hitting the Save/Update Record button will update the public site
```

Comment:
Put the > on the line above

Reply:
I dont know all of the ins and outs of spans, but I believe the reason for the > on the lower line is to ensure that intentional spaces that might come before the text aren't confused with the indentation spaces from the code formatting.